package.json

```diff
@@ -106,7 +106,7 @@
     "@shapeshiftoss/investor-yearn": "^6.1.2",
     "@shapeshiftoss/logger": "^1.1.3",
     "@shapeshiftoss/market-service": "^7.1.0",
-    "@shapeshiftoss/swapper": "^11.1.3",
+    "@shapeshiftoss/swapper": "~11.3.0",
```

Comment:
This one's correct, for this specific PR, we want only patch matching so we don't install 11.4.0 which comes with other breaking changes tackled separately.
This gets reverted to a carret version in the stacked PR
https://github.com/shapeshift/web/pull/2872/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R109